pkg/gadgets/trace/exec/tracer/bpf/execsnoop.bpf.c

```diff
@@ -41,7 +41,7 @@ struct {
 #ifdef WITH_LONG_PATHS
 	__uint(max_entries, 1024);
 #else /* !WITH_LONG_PATHS */
-	__uint(max_entries, 10240);
+	__uint(max_entries, 1024);
```

Comment:
The `#ifdef WITH_LONG_PATHS` line 41 could be removed since both branches are now the same.

Reply:
Oh wow lol :D